e2e/client/playwright/editor3.spec.ts

```diff
@@ -3,6 +3,48 @@ import {test, expect} from '@playwright/test';
 import {restoreDatabaseSnapshot, s} from './utils';
 import {getEditor3FormattingOptions, getEditor3Paragraphs} from './utils/editor3';
 import {TreeSelectDriver} from './utils/tree-select-driver';
+import {appConfig} from 'scripts/appConfig';
+
+test('can add embeds', async ({page}) => {
+    await restoreDatabaseSnapshot();
+
+    const monitoring = new Monitoring(page);
+
+    const requestRoute = 'https://sourcefabric.org';
+
+    await page.route(
+        `https://iframe.ly/api/oembed?callback=?&url=
+        ${requestRoute}
+        &api_key=${appConfig.iframely.key}
+        &omit_script=true&iframe=true`,
+        (route) => {
+            route.fulfill({
+                body: JSON.stringify([{
+                    title: 'Open Source Software for Journalism',
+                    // eslint-disable-next-line max-len
```

Comment:
if you did concatenate it - you long longer need to disable eslint for that line